posts/2020-11-27-typing-your-actors.adoc

```diff
@@ -0,0 +1,389 @@
+= Typing your actors
+Pedro Ferreira <pedro.ferreira@lunatech.nl>
```

Comment:
This should be pedrolunatech and not Pedro Ferreira <pedro.ferreira@lunatech.nl> . This is what I was trying to say last week @eloots .